doc/stdlib/collections.rst

```diff
@@ -1399,32 +1750,26 @@ Dequeues
 
 .. function:: push!(collection, items...) -> collection
 
-   Insert one or more ``items`` at the end of ``collection``.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       push!(collection, items...) → collection
 
-   .. doctest::
+   Insert ``items`` at the end of ``collection``.
 
-     julia> push!([1, 2, 3], 4, 5, 6)
```

Comment:
I think https://github.com/JuliaLang/julia/blob/181c19f5de971594fcd2b79dbf0789d159b579c2/base/docs/basedocs.jl#L318-L325 is overwriting the helpdb docstring. The helpdb version has a doctest which it would be good to keep.